src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -126,8 +126,9 @@ def encode_prompt(
         device: Optional[torch.device] = None,
         prompt_embeds: Optional[torch.FloatTensor] = None,
         negative_prompt_embeds: Optional[torch.FloatTensor] = None,
+        prompt_attention_mask: Optional[torch.FloatTensor] = None,
+        negative_prompt_attention_mask: Optional[torch.FloatTensor] = None,
         clean_caption: bool = False,
```

Comment:
```suggestion
        clean_caption: bool = False,
        **kwargs,
```

Reply:
Let's make sure to correctly deprecate `mask_feature`

Reply:
Done in https://github.com/huggingface/diffusers/pull/5752/commits/9e647b21200c64bdb3490fa20d0efdd50d968ebe. Also added a test in https://github.com/huggingface/diffusers/pull/5752/commits/277e1d08435ec5e5fac174adc40711e341a87fc7.

Reply:
Note that `encode_prompt` is also a public method